devfile.yaml

```diff
@@ -23,10 +23,8 @@ components:
       volumeMounts:
         - name: m2
           path: /home/user/.m2
-      memoryLimit: 6G
+      memoryLimit: 3G
```

Comment:
I found that the [native build task](https://github.com/che-incubator/quarkus-api-example/blob/992ad93649a36ef6845592c69a2650199d5a3a5d/devfile.yaml#L65-L72) works with `memoryLimit: 4G` however

Reply:
I think another option we can consider is remove the native package build task from the devfile cc @ibuziuk ? Since IMO the user-experience for this task was not good in the first place since it requires lots of memory, even the `6GB` was on the low side and caused some slowdowns

Reply:
Smoke tests passed with 4Gb https://github.com/che-incubator/che-code/actions/runs/5100692562/jobs/9169121340?pr=228

Reply:
2 devfiles is better yes.
For the future we need to discuss using another sample, that requires less resources than quarkus, for smoke testing.

Reply:
> For the future we need to discuss using another sample, that requires less resources than quarkus, for smoke testing.

I believe we should target smth. from https://registry.devfile.io/viewer

Reply:
Would it better to create a separate branch?
I have an empty workspace when using a devfile with different name.
I got nothing even if I add
```
starterProjects:
  - name: quarkus-api-example
    git:
      checkoutFrom:
        revision: main
      remotes:
        origin: https://github.com/che-incubator/quarkus-api-example.git
```